idaes/core/util/math.py

```diff
@@ -191,3 +191,16 @@ def safe_log(a, eps=1e-4):
         approximately log(max(a, eps))
     """
     return log(smooth_max(a, eps, eps=eps))
+
+
+def smooth_heaviside(x, k):
+    """
+    Provides a smooth, continuous approximation of a discontinuous function
+    Args:
+        x : Independent variable; as x increases, the function increases from 0 to 1
+        k : Smoothing parameter representing the slope of the discontinuity
+    Returns:
+        function : Continuous approximation of a discontinuous function
```

Comment:
Function which approximately equals 1 when x > 0 and approximately equals 0 when x < 0.